### PR TITLE
RF | POM - EntityArithmeticInline

### PR DESCRIPTION
**Pull request by MichaelMicrof**
[Trello card](https://trello.com/c/Kos50H2z/1162-rf-pom-entityarithmeticinline)

### CODE REVIEW

File: src/test/java/model/ArithmeticInlinePage.java [hunk 1]
```diff
@@ -34,10 +22,14 @@ public ArithmeticInlinePage(WebDriver driver) {
...
-        newButton.click();
+    @Override
+    protected ArithmeticInlineViewPage createViewPage() {
+        return null;
+    }
 
-        return new ArithmeticInlineEditPage(getDriver());
+    @Override
+    protected ArithmeticInlineEditPage createEditPage() {
+        return null;
```
Comment (ValeriiSarukhanov) [inline src/test/java/model/ArithmeticInlinePage.java:32]: return new ArithmeticInlineEditPage

File: src/test/java/model/ArithmeticInlinePage.java [hunk 2]
```diff
@@ -34,10 +22,14 @@ public ArithmeticInlinePage(WebDriver driver) {
         super(driver);
     }
 
-    public ArithmeticInlineEditPage clickNewButton() {
-        newButton.click();
+    @Override
+    protected ArithmeticInlineViewPage createViewPage() {
+        return null;
```
Comment (ValeriiSarukhanov) [inline src/test/java/model/ArithmeticInlinePage.java:27]: return new ArithmeticInlineViewPage

File: src/test/java/model/ArithmeticInlinePage.java (hunk 1 above)
Comment (MichaelMicrof) [reply src/test/java/model/ArithmeticInlinePage.java:32]: done

File: src/test/java/model/ArithmeticInlinePage.java (hunk 2 above)
Comment (MichaelMicrof) [reply src/test/java/model/ArithmeticInlinePage.java:27]: done

**Review by ValeriiSarukhanov — APPROVED** [review]

**Review by Wis13 — APPROVED** [review]
lgfm

**Pull request merged**